.github/workflows/test.yml

```
@@ -275,6 +275,13 @@ jobs:
           pip install --upgrade poetry
           poetry install --with dev --with test
       - run: cargo build -p test_util -vv # build scriptにより/crates/test_util/data/の生成
+      # 次の問題に対するワークアラウンド
+      # https://github.com/VOICEVOX/voicevox_core/issues/653#issuecomment-1782108410
+      - if: matrix.os == 'macos-latest'
```

他のとこと書き方を合わせるならこうかも？
```suggestion
      - if: startsWith(matrix.os, 'mac')
```